std_msgs/Header header
paul_vision/BBox2d[] boxes
# sensor_msgs/Image image
